src/sentry/static/sentry/app/components/alert.tsx

```diff
@@ -74,9 +74,10 @@ const IconWrapper = styled('span')`
   display: flex;
   margin-right: ${space(1)};
 
-  /* Minor adjustment to vertically align the icon, without using align-items
-   * which would vertically align it for multi-line alerts. */
-  padding-top: 1px;
+  /* Give the wrapper an explicit height so icons are line height with the
+   * (common) line height. */
+  height: 22px;
```

Comment:
`line-height` for Alerts is 1.5x so this should be `22.5px` :trollface: :trollface: :trollface:

https://github.com/getsentry/sentry/blob/f0401b2cca7702dfc9804c3b57b7ad195d34d23b/src/sentry/static/sentry/app/components/alert.tsx#L83

Reply:
go away

Reply:
😡

Reply:
😡 😡 😡 😡

Reply:
WOW disruptive content?! That's an A+ constructive comment 👨‍🎓